src/mesh.c

```
@@ -389,583 +368,295 @@ int mesh_update_routing(uint16_t source, uint16_t from_node, int16_t rssi, uint8
     return 0;
 }
 
-/**
- * Discover the network topology
- */
-int mesh_discover(void)
-{
-    /* Enter discovery mode */
+int mesh_discover(void) {
```

Cppcheck (2.21.0)

[style] The function 'mesh_discover' is never used.

(unusedFunction)